.github/workflows/lint.workflow.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  push:
+    branches:    
+      - '*'
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2

```

Comment:
You're totally right, sorry for the oversight. I've gone ahead and added it in this latest commit /w Node 18 (LTS). Let me know if you think that's good 😄.

Reply:
LTS is good. FYI; if our project needed `node` itself we'd have it specified in `package.json` `engines` (and we'd want to match it to that) but since we don't this is first pick and LTS is the natural choice.